openmrs/apps/ipdDashboard/app.json

```diff
@@ -0,0 +1,120 @@
+{
+  "sections": [
+    {
+      "title": "Vitals and Nutritional Values",
+      "componentKey": "VT",
+      "displayOrder": 1,
+      "refreshKey": 1
+    },
+    {
+      "title": "Allergies",
+      "componentKey": "AL",
+      "displayOrder": 2,
+      "refreshKey": 2
+    },
+    {
+      "title": "Diagnosis",
+      "componentKey": "DG",
+      "displayOrder": 3,
+      "refreshKey": 3
+    },
+    {
+      "title": "Treatments",
+      "componentKey": "TR",
+      "displayOrder": 4,
+      "refreshKey": 4
+    },
+    {
+      "title": "Nursing Tasks",
+      "componentKey": "NT",
+      "displayOrder": 5,
+      "refreshKey": 5
+    },
+    {
+      "title": "Drug Chart",
+      "componentKey": "DC",
+      "displayOrder": 6,
+      "refreshKey": 6
+    }
+  ],
+  "nursingTasks": {
+    "timeInMinutesFromNowToShowTaskAsRelevant": 30,
+    "timeInMinutesFromNowToShowPastTaskAsLate": 60,
+    "timeInMinutesFromStartTimeToShowAdministeredTaskAsLate": 60
+  },
+  "drugChart": {
+    "timeInMinutesFromNowToShowPastTaskAsLate": 60,
+    "timeInMinutesFromStartTimeToShowAdministeredTaskAsLate": 60
+  },
+  "drugChartSlider": {
+    "timeInMinutesToDisableSlotPostScheduledTime": 60
+  },
+  "enable24HourTime": false,
+  "medicationTags": {
+    "asNeeded": "Rx-PRN",
+    "STAT (Immediately)": "Rx-STAT",
```

Comment:
@Arjun-Go `STAT (Immediately)` I'm guessing is specific to CURE. Can you explain what this does?

Reply:
For nursing tasks tile and drug chart, we have tags showing up next to the medication name. So, any medication with frequency Immediately will have "Rx-STAT" inside the tag.
<img width="1792" alt="Screenshot 2024-06-06 at 9 25 17 AM" src="https://github.com/Bahmni/default-config/assets/91885483/e9cbc5e9-58ee-4d36-97ff-8dc30fae0340">